src/Symfony/Component/Notifier/Bridge/LineNotify/LineNotifyTransport.php

```diff
@@ -0,0 +1,104 @@
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Akira Kurozumi <info@a-zumi.net>
+ */
+final class LineNotifyTransport extends AbstractTransport
+{
+    protected const HOST = 'notify-api.line.me';
+
+    private const SUBJECT_LIMIT = 1000;
+
+    private string $token;
+
+    public function __construct(#[\SensitiveParameter] string $token, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
+    {
+        $this->token = $token;
+        $this->client = $client;
+        parent::__construct($client, $dispatcher);
+    }
+
+    protected function doSend(MessageInterface $message): SentMessage
+    {
+        if (!$message instanceof ChatMessage) {
+            throw new UnsupportedMessageTypeException(__CLASS__, ChatMessage::class, $message);
+        }
+
+        $content = $message->getSubject();
+
+        if (mb_strlen($content, 'UTF-8') > self::SUBJECT_LIMIT) {
+            throw new LengthException(sprintf('The subject length of a Line message must not exceed %d characters.', self::SUBJECT_LIMIT));
+        }
+
+        $endpoint = sprintf('https://%s/api/notify', $this->getEndpoint());
+        $response = $this->client->request('POST', $endpoint, [
+            'auth_bearer' => $this->token,
+            'query' => [
+                'message' => $content
+            ]
+        ]);
+
+        try {
+            $statusCode = $response->getStatusCode();
+        } catch (TransportException $e) {
+            throw new TransportException('Could not reach the remote Line server.', $response, 0, $e);
+        }
+
+        if (200 !== $statusCode) {
+            $result = $response->toArray(false);
+
+            if (401 === $statusCode) {
+                $originalContent = $message->getSubject();
+                $errorMessage = $result['message'];
+                $errorCode = $result['code'];
+                throw new TransportException(sprintf('Unable to post the Line message: "%s" (%d: "%s").', $originalContent, $errorCode, $errorMessage), $response);
+            }
+
+            if (400 === $statusCode) {
+                $originalContent = $message->getSubject();
+
+                $errorMessage = '';
+                foreach ($result as $fieldName => $message) {
+                    $message = \is_array($message) ? implode(' ', $message) : $message;
+                    $errorMessage .= $fieldName . ': ' . $message . ' ';
+                }
+            }
+
+            $errorMessage = trim($errorMessage);
+            throw new TransportException(sprintf('Unable to post the Line message: "%s" (%s).', $originalContent, $errorMessage), $response);
+        }
+
+        return new SentMessage($message, (string)$this);
+    }
+
+    public function supports(MessageInterface $message): bool
+    {
+        return $message instanceof ChatMessage;
+    }
+
+    public function __toString(): string
+    {
+        return sprintf('line-notify://%s', $this->getEndpoint());
```

Comment:
```suggestion
        return sprintf('linenotify://%s', $this->getEndpoint());
```
Let's not use a dash in the scheme